content/_partners/state-and-local._en.md

```diff
@@ -4,16 +4,9 @@ permalink: /partners/state-and-local/
 title: >-
   State, local, and territories
 body: >-
-  Crises, like the COVID-19 global pandemic and ongoing natural disasters, make it more urgent than ever for the public to gain fast and easy access to their benefits and services. The General Services Administration’s (GSA) Technology Transformation Services (TTS) realizes the burden this has put on state and local governments. Our team, with deep expertise in security, product, engineering, and user experience, is seeking state, local, and territory government partners that want to take advantage of all that Login.gov has to offer.
+  Crises, like the COVID-19 global pandemic and ongoing natural disasters, make it more urgent than ever for the public to gain fast and easy access to their benefits and services. The General Services Administration’s (GSA) Technology Transformation Services (TTS) realizes the burden this has put on state and local governments. Our team, with deep expertise in security, product, engineering, and user experience, is seeking state, local, and territory government partners who are supporting the implementation of a federal program at the state or local level. If you're unsure whether your project is eligible to use Login.gov, we encourage you to contact our partnerships team at partners@login.gov.
```

Comment:
Should `partners@login.gov` be linked?

Reply:
I think the `<>` syntax works with mailtos

```suggestion
  Crises, like the COVID-19 global pandemic and ongoing natural disasters, make it more urgent than ever for the public to gain fast and easy access to their benefits and services. The General Services Administration’s (GSA) Technology Transformation Services (TTS) realizes the burden this has put on state and local governments. Our team, with deep expertise in security, product, engineering, and user experience, is seeking state, local, and territory government partners who are supporting the implementation of a federal program at the state or local level. If you're unsure whether your project is eligible to use Login.gov, we encourage you to contact our partnerships team at <partners@login.gov>.
```